source/clear-linux/reference/collaboration/documentation/code.rst

```diff
@@ -3,7 +3,7 @@
 Code blocks
 ###########
 
-Collaborating to the |CLOSIA| is all about code. Therefore, your
+Collaborating to the |CL-ATTR| is all about code. Therefore, your
```

Comment:
Please change "Collaborating to" to "Contributing to" in Line 6.

Reply:
Done.